collections/src/main/java/com/couchbase/javaclient/doc/Emp.java

```diff
@@ -0,0 +1,138 @@
+    package com.couchbase.javaclient.doc;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Date;
+import java.util.Calendar;
+
+import com.couchbase.client.java.json.JsonObject;
+import com.github.javafaker.Faker;
+
+public class Emp implements DocTemplate{
+
+    JsonObject jsonObject = JsonObject.create();
+    private static final String[] FIRST_NAMES = {"Adara", "Adena", "Adrianne", "Alarice", "Alvita", "Amara",
```

Comment:
Do these names have to be exactly these values? Would prefer to use faker if these can be any value.

Reply:
Unfortunately, yes. I need to have same data for both test implementations: for buckets and for collections.